packages/vue-instantsearch/src/mixins/widget.js

```diff
@@ -31,10 +30,10 @@ export const createWidgetMixin = (
   created() {
     if (typeof connector === 'function') {
       this.factory = connector(this.updateState, () => {});
-      this.widget = _objectSpread(
-        this.factory(this.widgetParams),
-        additionalProperties
-      );
+      this.widget = {
+        ...this.factory(this.widgetParams),
+        ...additionalProperties,
+      };
```

Comment:
I believe this is what's causing the test failures. Using `Object.assign(this.factory(this.widgetParams), additionalProperties)` likely would pass; otherwise change the assertions to not be strict equal